server/lomas_server/dp_queries/dp_libraries/diffprivlib.py

```diff
@@ -9,23 +9,25 @@
     ExternalLibraryException,
     InternalServerException,
 )
+from lomas_core.models.requests import (
+    DiffPrivLibQueryModel,
+    DiffPrivLibRequestModel,
+)
+from lomas_core.models.responses import DiffPrivLibQueryResult
 from sklearn.model_selection import train_test_split
 from sklearn.pipeline import Pipeline
 
 from lomas_server.admin_database.admin_database import AdminDatabase
 from lomas_server.data_connector.data_connector import DataConnector
 from lomas_server.dp_queries.dp_libraries.utils import (
     handle_missing_data,
-    serialise_model,
 )
 from lomas_server.dp_queries.dp_querier import DPQuerier
-from lomas_server.models.requests import (
-    DiffPrivLibQueryModel,
-    DiffPrivLibRequestModel,
-)
 
 
-class DiffPrivLibQuerier(DPQuerier[DiffPrivLibRequestModel, DiffPrivLibQueryModel]):
+class DiffPrivLibQuerier(
+    DPQuerier[DiffPrivLibRequestModel, DiffPrivLibQueryModel, DiffPrivLibQueryResult]
```

Comment:
Why do we also need to specify DiffPrivLibQueryResult ?

Reply:
I think it comes from the changes made on the class DPQuerier. A QueryResultGeneric type variable was created in order to define the expected output. In that case, we therefore added DiffPrivLibQueryResult to indicate what is the expected output here (for DPQUerier)

Reply:
ok ok